modules/lazyload/module.php

```diff
@@ -54,6 +54,13 @@ private function update_element_attributes( Element_Base $element ) {
 
 		if ( $keys ) {
 			$background_image_url = Utils::get_array_value_by_keys( $settings, $keys );
+			if ( ! $background_image_url ) {
+				// If the background image is not set, check also for overlay image.
+				$keys = Utils::get_array_value_by_keys( $control_data, [ 'background_lazyload', 'overlay_keys' ] );
+				if ( $keys ) {
+					$background_image_url = Utils::get_array_value_by_keys( $settings, $keys );
+				}
+			}
```

Comment:
Add a space below the condition and consider adding spaces in general for readability (to keep on a consistent code style in the project).

Reply:
My PHPCS is not allowing me to add space below the condition.
The empty lines are auto removed when I save the file.
I'm using our config file `phpcs.xml`

Reply:
PHPCS error is generated by the line break in both my local and the actual PR test.